src/critnib/critnib.c

```diff
@@ -399,9 +446,14 @@ int critnib_insert(struct critnib *c, word key, void *value, int update) {
 /*
  * critnib_remove -- delete a key from the critnib structure, return its value
  */
-void *critnib_remove(struct critnib *c, word key) {
+void *critnib_remove(struct critnib *c, word key, void **ref) {
```

Comment:
add info about "ref" to the comment above

Reply:
Done

Reply:
Done